jwt.el

```diff
@@ -613,13 +613,58 @@ Specifically it checks that TEST-STRING has
            (doco (car maybe-doc)))
       (funcall callback doco :thing full-name))))
 
+(defvar jwt--annotation-claims
+  '(("nbf" . "Not before")
+    ("iat" . "Issued at")
+    ("exp" . "Expires")))
+
+(defface jwt-annotation
+  '((t
+     (:box
+      (:line-width (1 . 1)
+                   :color nil
+                   :style nil)
+      :inherit (magit-branch-local))))
+  "Face for JWT claim annotation overlays.")
+
+(defun jwt--annotation-add-overlays (beg end)
+  "Add JWT related overlays between BEG and END."
+  (save-excursion
+    (goto-char beg)
+    (let ((claims-rx (rx "\"" (eval (cons 'group (list (cons 'or (mapcar #'car jwt--annotation-claims))))) "\""
+                         (* space) ":" (* space) (group (+ digit)))))
+      (while (re-search-forward claims-rx end t)
+        (let ((ov (make-overlay (match-beginning 0) (match-end 0)))
+              (claim (match-string-no-properties 1))
+              (formatted-time (thread-last (match-string-no-properties 2)
+                                           string-to-number
+                                           seconds-to-time
```

Comment:
`format-time-string` assumes a number argument is seconds anyway, so you don't need `seconds-to-time` here